Src/FluentAssertions/Equivalency/Selection/SelectMemberByPathSelectionRule.cs

```diff
@@ -16,6 +16,11 @@ protected SelectMemberByPathSelectionRule(string selectedPath)
 
         public virtual bool IncludesMembers => false;
 
+        protected string SelectedPath
```

Comment:
🤔 We prefer to use a method instead of a write-only property

Reply:
✅ New method `SetSelectedPath`